app/javascript/mastodon/features/ui/components/share_profile_modal.js

```diff
@@ -0,0 +1,191 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ImmutablePureComponent from 'react-immutable-pure-component';
+import IconButton from '../../../components/icon_button';
+import { defineMessages, FormattedMessage, injectIntl } from 'react-intl';
+import Icon from '../../../components/icon';
+
+const messages = defineMessages({
+  close: { id: 'lightbox.close', defaultMessage: 'Close' },
+  twitterText: { id: 'profile.share.twitter_text', defaultMessage: 'I\'m {handle} on Mastodon, follow me:' },
+  twitterHashtag: { id: 'profile.share.twitter_hashtag', defaultMessage: 'ImOnMastodon' },
+  shareFacebook: { id: 'profile.share.share_facebook', defaultMessage: 'Share on Facebook' },
+  shareTwitter: { id: 'profile.share.share_twitter', defaultMessage: 'Share on Twitter' },
+  shareVK: { id: 'profile.share.share_vk', defaultMessage: 'Share on VK' },
+  shareClipboard: { id: 'profile.share.share_clipboard', defaultMessage: 'Copy to clipboard' },
```

Comment:
I think of changing this to “Copy URL to clipboard”, but at the same time I like current variant because it's simpler.